### PR TITLE
Use LSP4J 0.21

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

File: target-platforms/target-platform-latest/target-platform-latest.target [hunk 1]
```diff
@@ -15,7 +15,7 @@
 			<unit id="org.eclipse.lsp4j.jsonrpc.debug" version="0.0.0"/>
 			<unit id="com.google.gson" version="0.0.0"/>
 			<unit id="com.google.guava" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.20.1/"/>
+			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.21.0/"/>
```
Comment (cdietrich) [inline target-platforms/target-platform-latest/target-platform-latest.target:18]: depending what you do in your codebase, you can remove xbase (and guava) now

Comment (mickaelistria) [reply target-platforms/target-platform-latest/target-platform-latest.target:17]: Awesone, thanks for the heads up!

Comment (cdietrich) [reply target-platforms/target-platform-latest/target-platform-latest.target:17]: looks like guava is still used in a few places in lsp4e codebase

**Comment by mickaelistria** [comment]
The dash license issue can be ignored now; as we're removing deps and not adding any.

**Pull request merged**